lighthouse-cli/compiled-check.js

```diff
@@ -6,7 +6,7 @@ const path = require('path');
 module.exports = function(filename) {
   if (!fs.existsSync(path.join(__dirname, filename))) {
     console.log('Oops! Looks like the CLI needs to be compiled. Please run:');
-    console.log('   cd lighthouse-cli && npm install && npm run build');
+    console.log('   cd lighthouse-cli && npm install');
```

Comment:
wouldn't we not see this anymore, as all installs (either `npm install lighthouse` (which has these files included) and `npm install` inside the cloned repo) will have the compiled files?